explorer/interpreter/resolve_unformed.cpp

```diff
@@ -293,6 +293,15 @@ static auto ResolveUnformed(Nonnull<const Declaration*> declaration)
       }
       break;
     }
+    case DeclarationKind::DestructorDeclaration: {
+      auto& destructor = cast<DestructorDeclaration>(*declaration);
```

Comment:
```suggestion
    case DeclarationKind::FunctionDeclaration:
    case DeclarationKind::DestructorDeclaration: {
      auto& destructor = cast<CallableDeclaration>(*declaration);
```

Can this be combined with the above `case`?